BUILD.gn

```diff
@@ -54,55 +54,40 @@ config("branding") {
 }
 
 npm_action("atom_browserify_sandbox") {
-  deps = [
-    ":atom_js2c_copy",
-  ]
+  script = "browserify"
 
-  sandbox_args = [
-    "lib/sandboxed_renderer/init.js",
-    "-r",
-    "./lib/sandboxed_renderer/api/exports/electron.js:electron",
-    "-t",
-    "aliasify",
+  outputs = [
+    "$target_gen_dir/js2c/preload_bundle.js",
   ]
 
-  inputs = [
+  args = [
     # FIXME(zcbenz): The dependencies of these files are not listed here, so
     # the generated file will be out-dated when dependencies are modified.
     # Use a script to generate all dependencies and put them here.
     "lib/sandboxed_renderer/init.js",
-    "lib/sandboxed_renderer/api/exports/electron.js",
-  ]
-  outputs = [
-    "$target_gen_dir/js2c/preload_bundle.js",
+    "-r",
+    "./lib/sandboxed_renderer/api/exports/electron.js:electron",
+    "-t",
+    "aliasify",
+    "-o",
+    rebase_path(outputs[0]),
   ]
-
-  script = "browserify"
-  args = sandbox_args + [
-           "-o",
-           rebase_path(outputs[0]),
-         ]
 }
 
 npm_action("atom_browserify_isolated") {
-  deps = [
-    ":atom_js2c_copy",
-  ]
+  script = "browserify"
 
-  inputs = [
-    "lib/isolated_renderer/init.js",
-  ]
   outputs = [
     "$target_gen_dir/js2c/isolated_bundle.js",
   ]
 
-  script = "browserify"
-  args = inputs + [
-           "-t",
-           "aliasify",
-           "-o",
-           rebase_path(outputs[0]),
-         ]
+  args = [
```

Comment:
here too, `inputs` is a special GN name and not just fluff. without it GN won't know when it needs to rebuild this target.